api-reference/arcana/streaming-pcm.mdx

```diff
@@ -39,6 +39,10 @@ authMethod: "bearer"
   Float that controls the cumulative probability of the top tokens to consider. Must be in (0, 1]. Set to 1 to consider all tokens.
 </ParamField>
 
+<ParamField body="sampling_rate" type="int" default="24000">
+  The sampling rate (Hz). Range is 8000 to 48000.
```

Comment:
I'll need one of the ML folks to clarify about sampling rate values. My understanding is that only _specific_ values between those ranges are supported. But I could be wrong!